File: app/src/external/java/com/kickstarter/libs/NoopBuildCheck.java
package com.kickstarter.libs;

import androidx.annotation.NonNull;

import com.kickstarter.services.WebClientType;
import com.kickstarter.viewmodels.DiscoveryViewModel;

public class NoopBuildCheck implements BuildCheck {
  @Override
  public void bind(final @NonNull DiscoveryViewModel.ViewModel viewModel, final @NonNull WebClientType client) {
    // No-op, distribution through Play Store
  }
}

File: app/src/regression/java/com/kickstarter/libs/NoopBuildCheck.kt
package com.kickstarter.libs

import androidx.annotation.NonNull
import com.kickstarter.services.WebClientType
import com.kickstarter.viewmodels.DiscoveryViewModel

class NoopBuildCheck : BuildCheck {
    @Override
    override fun bind(@NonNull viewModel: DiscoveryViewModel.ViewModel, @NonNull client: WebClientType) {
        // No-op
    }
}
